mscore/inspector/inspectorTextLineBase.cpp

```diff
@@ -223,6 +223,7 @@ void InspectorTextLineBase::hasBeginTextClicked(bool checked)
             t->undoResetProperty(Pid::BEGIN_FONT_SIZE);
             t->undoResetProperty(Pid::BEGIN_FONT_STYLE);
             t->undoResetProperty(Pid::BEGIN_TEXT_OFFSET);
+            tl.beginText->clear();
```

Comment:
These used to be cleared by way of a new InspectorTextLineBase construction in... iirc ::update().

Reply:
That's to say:
1) When having for example "Begin Text" checked, yet having nothing in the text edit: if the user deselects the inspected item, then reselects it, Begin Text will be unchecked. 
2) Likewise if text exists, but then the user unchecks "Begin Text", that text gets cleared. Yet it gets cleared not by some specific clearing mechanism for the text, but rather a construction/destruction of the inspector object occurs, getting its default state. 

Why is this relevant? Because the function `setElement() `"reinvigorates" a new Inspector object, and it is invoked for some weird reason when initially clicking into the text-edit box after having enabled, for example, "Begin text". The "loss of focus" is in actuality a re-construction of the entire freakin' inspector object. 

The changes here stop that re-construction with the verification that there is a "different" element (kind of like what the check is supposed to be doing with 

```cpp
void Inspector::update(Score* s)
      {
      if (_inspectorEdit)     // if within an inspector-originated edit
            return;
```
... but then there's needed a manual clearing of the text so that there isn't any dormant text in the edit-box due to no longer reconstruction of the inspector element. 

_End of needless explanation to self for the purposes of "I know not what"_